lib/ws-kernel.coffee

```diff
@@ -18,8 +18,14 @@ class WSKernel extends Kernel
     interrupt: ->
         @session.kernel.interrupt()
 
+    shutdown: (restart = false) ->
+        if restart
```

Comment:
After re-reading the jupyter messaging spec, it looks like you're right.

I'll remove this handling here to avoid confusion